smartsim/_core/_cli/build.py

```diff
@@ -399,18 +421,22 @@ def check_tf_install(self) -> None:
                 "tensorflow", self.versions.TENSORFLOW
             ):
                 msg = (
-                    f"TensorFlow {self.versions.TENSORFLOW} not installed in Python environment. "
-                    + f"Consider installing tensorflow=={self.versions.TENSORFLOW} with pip"
+                    "TensorFlow %s not installed in Python environment. "
+                    + "Consider installing tensorflow==%s with pip",
+                    self.versions.TENSORFLOW,
+                    self.versions.TENSORFLOW,
                 )
                 logger.warning(msg)
             else:
                 logger.info(
-                    f"TensorFlow {self.versions.TENSORFLOW} installed in Python environment"
+                    "TensorFlow {} installed in Python environment",
+                    self.versions.TENSORFLOW,
                 )
         except SetupError as e:
             logger.warning(str(e))
 
-    def check_backends_install(self) -> None:
+    @staticmethod
```

Comment:
You made this a `staticmethod` because `self` is never accessed?

Reply:
pylint dings anything that doesn't use self. I can disable that check if you wanted to avoid the statics.